web/app/frontend/src/components/form/FormControl.tsx

```diff
@@ -8,51 +8,49 @@ import ReferenceInput from "./ReferenceInput";
 
 // TODO: move & clean up types
 export type InputProps = {
-    fieldDefinition: FieldDefinition,
-    style?: ViewStyle,
-    value: any,
-    onChange: any,
-    onBlur?: any,
-    error?: any,
-    invalid?: boolean,
-    isTouched?: boolean,
-    isDirty?: boolean,
-    isMultiple?: boolean,
-    isQuantity?: boolean
+    fieldDefinition: FieldDefinition;
+    style?: ViewStyle;
+    value: any;
+    onChange: any;
+    onBlur?: any;
+    error?: any;
+    invalid?: boolean;
+    isTouched?: boolean;
+    isDirty?: boolean;
+    isMultiple?: boolean;
+    isQuantity?: boolean;
 };
 
 type FormControlProps = {
-    name: string,
-    fieldDefinition: FieldDefinition,
-    style?: ViewStyle,
-    value?: any,
-    control: Control<any, object>,
-    errors?: object
+    name: string;
+    fieldDefinition: FieldDefinition;
+    style?: ViewStyle;
+    value?: any;
+    control: Control<any, object>;
+    errors?: object;
+    testID?: string;
```

Comment:
since this is always "formControl", I don't think it needs to be passed as a parameter

Reply:
I believe we may need it because it is being consumed by the Controller component within the FormControl stack

https://react-hook-form.com/api/useform/control

Reply:
yes, it's needed, but it doesn't need to be a parameter because it doesn't change based on the field type. you can just put the constant in line 44 here.

Reply:
I'm not sure I understand. Do you mean `const { control } = useForm()`?

Reply:
I mean you can just import the testIDs in this file instead of in the AddRecordScreen and use them in the line `<View style={{ margin: 10 }} testID={testIds.formControl}>`

Reply:
ah, yes you are correct, good catch